### ISSUE TITLE
last versions of Opera generates empty regexp

### ISSUE THREAD

**Issue by DarthVitalus** [browserslist/browserslist-useragent-regexp#926]
Hi! First of all - AWESOME tool, guys! Thank you so much for it!

Now to unexpected behavior :)

**Code sample:**

```
const {
    getUserAgentRegExp,
    getUserAgentRegExps,
} = require('browserslist-useragent-regexp');


const browserListConfig = {
    browsers: 'last 2 Opera versions',
};

console.log(getUserAgentRegExps(browserListConfig));
console.log(getUserAgentRegExp(browserListConfig));
```

**Expected Behavior:** 
regexes and regexp objects as for other family (like Chrome)

**Actual behavior:**
```
[]
/(?:)/
```

**Comment by DarthVitalus** [comment]
Wow, that was quick 😅💪

**Comment by dangreen** [comment]
@DarthVitalus Hi. Thank you for the issue. Will make release with fix later today.

**Comment by dangreen** [comment]
@DarthVitalus Fixed in [v3.0.2](https://www.npmjs.com/package/browserslist-useragent-regexp/v/3.0.2)
